docs/notebooks/image_to_image_arcface.ipynb

```diff
@@ -0,0 +1,362 @@
+        "INFO     Done ✨                                                                              __main__.py:195\n",
+        "DEBUG    Finetuning took 0 days, 0 hours 3 minutes and 48 seconds                             __main__.py:197\n",
+        "INFO     Metric: 'resnet50_precision_at_k' before fine-tuning:  0.11575 after fine-tuning:    __main__.py:210\n",
+        "0.53425\n",
+        "INFO     Metric: 'resnet50_recall_at_k' before fine-tuning:  0.05745 after fine-tuning:       __main__.py:210\n",
+        "0.27113\n",
+        "INFO     Metric: 'resnet50_f1_score_at_k' before fine-tuning:  0.07631 after fine-tuning:     __main__.py:210\n",
+        "0.35788\n",
+        "INFO     Metric: 'resnet50_hit_at_k' before fine-tuning:  0.82900 after fine-tuning: 0.94100  __main__.py:210\n",
+        "INFO     Metric: 'resnet50_average_precision' before fine-tuning:  0.52305 after fine-tuning: __main__.py:210\n",
+        "0.79779\n",
+        "INFO     Metric: 'resnet50_reciprocal_rank' before fine-tuning:  0.64909 after fine-tuning:   __main__.py:210\n",
+        "0.89224\n",
+        "INFO     Metric: 'resnet50_dcg_at_k' before fine-tuning:  1.30710 after fine-tuning: 4.52143  __main__.py:210\n",
+        "INFO     Building the artifact ...                                                            __main__.py:215\n",
+        "INFO     Pushing artifact to Jina AI Cloud ...                                                __main__.py:241\n",
+        "[12:19:53] INFO     Artifact pushed under ID '63f8a9089c6406e19244771d'                                  __main__.py:243\n",
+        "DEBUG    Artifact size is 83.580 MB                                                           __main__.py:245\n",
+        "INFO     Finished 🚀                                                                          __main__.py:246\n",
+        "```"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0l4e4GrspilM"
+      },
+      "source": [
+        "## Saving\n",
+        "\n",
+        "After the run has finished successfully, you can download the tuned model on your local machine:\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "KzfxhqeCmCa8"
+      },
+      "outputs": [],
+      "source": [
+        "artifact = run.save_artifact('resnet-model')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gkNHTyBkprQ0"
+      },
+      "source": [
+        "## Inference\n",
+        "\n",
+        "Now you saved the `artifact` into your host machine,\n",
+        "let's use the fine-tuned model to encode a new `Document`:\n",
+        "\n",
+        "```{admonition} Inference with ONNX\n",
+        "In case you set `to_onnx=True` when calling `finetuner.fit` function,\n",
+        "please use `model = finetuner.get_model(artifact, is_onnx=True)`\n",
```

Comment:
```suggestion
        "Now that you've saved the `artifact` into your host machine,\n",
        "let's use the fine-tuned model to encode a new `Document`:\n",
        "\n",
        "```{admonition} Inference with ONNX\n",
        "If you set `to_onnx=True` when calling the `finetuner.fit` function,\n",
        "please use `model = finetuner.get_model(artifact, is_onnx=True)`\n",
```